cabal-install/src/Distribution/Client/Init/Interactive/Command.hs

```diff
@@ -276,10 +276,16 @@ genTestTarget flags pkgs = initializeTestSuitePrompt flags >>= go
 
 overwritePrompt :: Interactive m => InitFlags -> m Bool
 overwritePrompt flags = do
-  isOverwrite <- getOverwrite flags
-  promptYesNo
-    "Do you wish to overwrite existing files (backups will be created) (y/n)"
-    (DefaultPrompt isOverwrite)
+  con <- listDirectory $ fromFlagOrDefault "." (packageDir flags)
```

Comment:
You might want to use `getPackageDir` here, for consistency
https://github.com/haskell/cabal/blob/a40876dbdb0fe624cb8c64453ece3c768a79832c/cabal-install/src/Distribution/Client/Init/FlagExtractors.hs#L68-L69